types/sinon/index.d.ts

```diff
@@ -11,7 +11,7 @@
 //                 Simon Schick <https://github.com/SimonSchick>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-import * as FakeTimers from '@sinonjs/fake-timers';
+import * as FakeTimers from "@sinonjs/fake-timers";
```

Comment:
shouldn't this need a package.json now to install the real `@sinonjs/fake-timers`?

Reply:
i think i must've just forgot to check it in, i do have one locally :D ill sort it out 👍